app/variablesmanager.h

```diff
@@ -0,0 +1,24 @@
+#ifndef VARIABLESMANAGER_H
+#define VARIABLESMANAGER_H
+
+#include <QObject>
+#include "merginapi.h"
+#include "qgsproject.h"
+
+class MerginApi;
+
+class VariablesManager : public QObject
```

Comment:
Could you please add doxygen comments to the class about the variables that it sets?